app/assets/javascripts/controllers/container_project/container_project_controller.js

```diff
@@ -0,0 +1,39 @@
+angular.module('containerProjectDashboard', ['ui.bootstrap', 'patternfly', 'patternfly.charts', 'containerDashboard'])
+    .config(['$httpProvider', function($httpProvider) {
+        $httpProvider.defaults.headers.common['X-CSRF-Token'] = jQuery('meta[name=csrf-token]').attr('content');
+    }])
+    .controller('containerProjectDashboardController', ['$scope','ContainerDashboardUtils', '$http', '$interval', "$location",
+        function($scope, containerDashboardUtils, $http, $interval, $location) {
+            document.getElementById("center_div").className += " miq-body";
+
+            $scope.objectStatus = {
+                nodes:       containerDashboardUtils.createNodesStatus(),
```

Comment:
this area, as well as the "updateStatus" one below, are going to be repeated a lot through the entities. 
it's worth to think how to reuse code and avoid copy paste. perhaps create a method that receives list of entities, and then loop through them in a more generic way.